example/plugin-example/secrets.nix

```diff
@@ -0,0 +1,7 @@
+let
```

Comment:
Maybe just move this to `example/` and name it `secrets-plugin.nix`.

Reply:
Done.